.github/pull_request_template.md

```diff
@@ -20,6 +20,8 @@
 
 -   [ ] I have added the major changes included in this pull request to the `NEWS` under the next release version or unreleased, and updated the date.
 
+-   [ ] I have updated the version number in `DESCRIPTION` according to [semantic versioning](https://semver.org/) rules.
```

Comment:
Hate to be pedantic, but I wonder if we should combine these for clarity. I see it all as one step, as the date and version number should be consistent between `NEWS` and `DESCRIPTION`. I'm hoping that we can teach new contributors through this checklist. If we get this right, we'd avoid a lot of PR review comments.

Reply:
I merged the NEWS and DESCRIPTION lines